src/renderer/components/+apps-releases/release-rollback-dialog.tsx

```diff
@@ -16,24 +16,31 @@ interface Props extends DialogProps {
 
 @observer
 export class ReleaseRollbackDialog extends React.Component<Props> {
-  @observable static isOpen = false;
-  @observable.ref static release: HelmRelease = null;
+  static metadata = observable({
+    isOpen: false,
+    release: null as HelmRelease,
+  });
```

Comment:
Isn't `observable` generic over its input? Then this could be:


```suggestion
interface Metadata {
  isOpen: boolean;
  release: HelmRelease | null;
}

...

  static metadata = observable<Metadata>({
    isOpen: false,
    release: null,
  });
```